api/db/postgres/postgres_handler.go

```diff
@@ -0,0 +1,48 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"sync"
+	"time"
+	_ "github.com/lib/pq" // Defining postgres plugin
+)
+
+var (
+	doOnce sync.Once
+	db     *sql.DB = nil
+	dbErr  error   = nil
+)
+
+// ConfigureDB will establish a new connection
+// with the postgres DB referenced in the arguments
+func (pHandler *PostgresHandler) ConfigureDB(
+	address string,
+	username string,
+	password string,
+	dbName string) error {
+	doOnce.Do(func() {
+		connStr := fmt.Sprintf("postgres://%s:%s@%s/%s?sslmode=disable",
```

Comment:
What do you think of adding `sslmode` as a configurable option as well? By the way, are we expecting this `disable` to be the default value? 🧐

Reply:
Absolutely!